arviz/tests/test_stats.py

```diff
@@ -140,8 +140,13 @@ def test_summary_unpack_order(order):
         assert col1 == col2
 
 
-def test_summary_stat_func(centered_eight):
-    assert summary(centered_eight, stat_funcs=[np.var]) is not None
+@pytest.mark.parametrize(
+    "stat_funcs", [[np.var], {"var": np.var, "var2": lambda x: np.var(x) ** 2}]
```

Comment:
In regards to the comment above this test seems to pass a list and still functions.

Reply:
Yes, I wanted to keep original structure